extension/main.js

```diff
@@ -85,18 +85,21 @@
 
 		const ghTab = {url};
 
-		if (window.GitHubNotify.settings.get('count') > 0) {
-			ghTab.url = `${url}notifications`;
-			if (window.GitHubNotify.settings.get('useParticipatingCount')) {
-				ghTab.url += '/participating';
-			}
+		
+		ghTab.url = `${url}notifications`;
+		if (window.GitHubNotify.settings.get('useParticipatingCount')) {
+			ghTab.url += '/participating';
 		}
 
-		if (typeof tab !== 'undefined' && (tab.url === '' || tab.url === 'chrome://newtab/' || tab.url === ghTab.url)) {
```

Comment:
Which logic? 'chrome://newtab/' gets replaced instead of a new tab? What does the 'undefined' check give us here? Thanks.